pkg/cli/image/manifest/manifest.go

```diff
@@ -276,47 +276,78 @@ func FirstManifest(ctx context.Context, from imagereference.DockerImageReference
 }
 
 // ManifestToImageConfig takes an image manifest and converts it into a structured object.
-func ManifestToImageConfig(ctx context.Context, srcManifest distribution.Manifest, blobs distribution.BlobService, location ManifestLocation) (*dockerv1client.DockerImageConfig, []distribution.Descriptor, error) {
+func ManifestToImageConfig(ctx context.Context, srcManifest distribution.Manifest, blobs distribution.BlobService, manifests distribution.ManifestService, location ManifestLocation, platformSpec string) (*dockerv1client.DockerImageConfig, []distribution.Descriptor, string, error) {
```

Comment:
You need to document using variable names in the signature what the new return value actually is.  Also improve the godoc.

Reply:
The godoc for instance needs to describe what this method does with its inputs.

Reply:
updated